tests/s3_data_plane_tests.c

```diff
@@ -8114,3 +8114,52 @@ static int s_test_s3_default_get_without_content_length(struct aws_allocator *al
 
     return AWS_OP_SUCCESS;
 }
+
+struct aws_string *test_helper(struct aws_allocator *allocator) {
```

Comment:
This helper function is defined in the middle of the test file rather than at the top with other helper functions. Consider moving it to be grouped with other similar helper functions for better code organization.